production/loki-mixin/dashboards/dashboard-loki-operational.json

```diff
@@ -2543,9 +2543,9 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{cluster=\"$cluster\", namespace=\"$namespace\", pod=~\"ingester.*\"}",
+              "expr": "sum by(pod) (rate(container_cpu_usage_seconds_total{cluster=~\"$cluster\", namespace=~\"$namespace\", container=\"ingester\"}[$__rate_interval]))",
```

Comment:
Nice; this has the added benefit of not relying on a recording rule which may not be present in the user's environment.

Reply:
Note that recording rule joins `node` in; if you need that then it's expensive to recompute it on the fly.

However there was no sign that you needed it.